config/database.yml.example

```diff
@@ -1,8 +1,37 @@
+#########################################################################
 # SQLite version 3.x
 #   gem install sqlite3
 #
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
+#   The configuration for SQLite has been done already.
+##########################################################################
+# PostgreSQL Configuration:
+#   Execute
+#   gem install pg
+#   Ensure the PostgreSQL 'pg' gem is defined in your Gemfile
+#   gem 'pg'
+#   For PostgreSQL to work set the adapter below in development,
+#   test and production environment as follows. You need to create role for
+#   your app to access the database.
+#
+#
+#   EXAMPLE:
+#
+#   development:
+#      adapter: postgresql
+#      database: your_development_database_name
+#      username: role_username
+#      password: role_password
+#
+#   The adapters for development,test and production need to be overridden
```

Comment:
I am not sure why we need this part here, to remind people they need to remove/comment out the existing configuration? Perhaps we should mention on the top that they can choose which db they want, and to use whichever config they need. 

We have sqlite3 uncommented everywhere, and then pg and mysql commented out.

Reply:
okay